src/SFA.DAS.EmployerAccounts.Web.UnitTests/Controllers/EmployerTeamControllerTests/WhenCallToActionToggleIsEnabled/WhenFundsToReserve.cs

```diff
@@ -10,7 +10,6 @@
 using SFA.DAS.EmployerAccounts.Web.Controllers;
 using SFA.DAS.EmployerAccounts.Web.Orchestrators;
```

Comment:
namespace cleanup

Reply:
There seems to be 2 files with the same name. Can you please check if its a duplicate?

Reply:
yep. one for CallToAction.